src/main/java/core/basesyntax/service/impl/ValidatorImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Validator;
+import java.util.List;
+
+public class ValidatorImpl implements Validator {
+    private static final int INDEX_OF_OPERATION = 0;
+    private static final int INDEX_OF_FRUIT = 1;
+    private static final int INDEX_OF_QUANTITY = 2;
+
+    @Override
+    public boolean validate(List<String> data) {
```

Comment:
I think we need more checks for validate such as for numbers in fruitName or some signs